backend/tenant/tenant.py

```diff
@@ -137,3 +137,38 @@ def get_all_offices(self):
             'success': True,
             'offices': cursor.fetchall()
         }
+
+    def searchOffices(self, filter):
```

Comment:
Você vai precisar verificar se cada uma das keys é valida pra adicionar filtro ou não, tô trabalhando pra enviar `None` no valor se não for válida.